info/available-rules.md

```diff
@@ -1,131 +1,131 @@
+| 1    | 1.1.1    | FILE_NAME_INCORRECT                       | Check: warns if a file name does not have an extension .kt/.kts<br>Fix: no                                                                                                                                                                                        | no  | no                                                                                                                                                                                                                         | Extensions should be configurable; it can be autofixed                                                                                                                                                             |
+| 1    | 1.1.1    | GENERIC_NAME                              | Check: warns if a generic name contains more than 1 letter (capital). It can be followed by numbers, for example: T12, T<br>Fix:                                                                                                                                   | yes | no                                                                                                                                                                                                                         | Recursively update usages of this identifier in the project                                                                                                                                                                   |
+| 1    | 1.1.1    | VARIABLE_HAS_PREFIX                       | Check: warns if a variable has a prefix (like mVariable or M_VARIABLE), generally it is a bad code style (Android - is the only exception<br>Fix: no fix is available to automatically create a name                                                      | no  | no                                                                                                                                                                                                                         | Maybe we can fix it, but I do not see any sense for it                                                                                                                                                                      |
```

Comment:
Check: warns if **the** name of **a** variable has a [Hungarian notation](https://en.wikipedia.org/wiki/Hungarian_notation) specific prefix (like `mVariable` or `M_VARIABLE`); generally, it is **considered** a bad code style (_Android_ **being** the the only exception).

Fix: none is available, as only a human can choose a meaningful name depending on the context.

Reply:
Question: Do we need the following?
generally, it is **considered** a bad code style (_Android_ **being** the the only exception).

Reply:
Do you mean the sentence per se? Or the formatting?

The formatting is mine, merely to emphasize the additions to the text. Being just an emphasis, the formatting should not propagate into the final document.

Speaking of the sentence -- yes, I do want this exact wording (possibly with your minor corrections) into the final proofread version.

Reply:
OK. I changed it to read:

Check: warns if the name of a variable has a [Hungarian notation](https://en.wikipedia.org/wiki/Hungarian_notation) specific prefix (such as `mVariable` or `M_VARIABLE`), which is considered a bad code style (_Android_ is the only exception).